src/styles/variables.scss

```diff
@@ -1,4 +1,5 @@
 .Layer__component,
+.Layer__Portal,
```

Comment:
Help me understand the distinction. Are portals a strict subset of components? 

My intuition of the concepts, tell me where im wrong: A top-level "component" a customer directly uses would be a component _and_ a portal whereas something like a button might be exposed as a component (to be customizable) but not a portal because it wouldn't be rendered directly, only as part of another component.

Reply:
Agreed that a portal is a subset of a component.

I would like our codebase to **not** be littered with references to `Layer__component`. Only pieces directly accessible in the public API surface should have this class name. Inheritance should ensure that everything rendered underneath these pieces has the desired styling.

Portal is the one special case where we cannot rely on a cascade, since they are appended somewhere much higher up in the DOM.